lib/main-menu.js

```diff
@@ -153,6 +153,19 @@ var view = {
   label: 'View',
   submenu: [
     {
+      label: 'Toggle Full Screen',
+      accelerator: (() => {
+        if (process.platform === 'darwin') return 'Ctrl+Command+F'
+        return 'F11'
+      })(),
+      click: (item, focusedWindow) => {
```

Comment:
Got it :smile: So could you add a comment like `// item is not used` under L161?